pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -531,7 +532,14 @@ def _save_last_checkpoint(self, trainer, pl_module, ckpt_name_metrics, filepath)
             )
             last_filepath = os.path.join(self.dirpath, f"{last_filepath}.ckpt")
 
-        self._save_model(last_filepath, trainer, pl_module)
+        accelerator_backend = trainer.accelerator_backend
+
+        ddp_plugin = accelerator_backend.ddp_plugin if accelerator_backend is not None else None
+        if ddp_plugin is not None and ddp_plugin.using_rpc_async:
```

Comment:
Can we ensure that this is only called if it's an rpc plugin? `isinstance(ddp_plugin, RPCPlugin)`

Reply:
I tried, but there is the wierd bug with the doc. Not idea why it is happening.